exercises/practice/kindergarten-garden/.meta/example.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <array>
+#include <string_view>
+
+namespace kindergarten_garden {
+
+enum class Plants { grass = 'G', clover = 'C', radishes = 'R', violets = 'V' };
```

Comment:
```suggestion
enum class Plants : char { grass = 'G', clover = 'C', radishes = 'R', violets = 'V' };
```

Reply:
Too bad I still don't get an implicit cast with this. I could skip to loc.

Reply:
True. But there's no such thing as implicit conversion from underlying to enum. You could cast the input array as enums, but I wouldn't consider that very clean. There's one other thing you could do to make the code.a bit more compact, though. Let me add another comment there.